Example/Auth/Tests/FIRAuthAppDelegateProxyTests.m

```diff
@@ -312,6 +427,16 @@ - (void)testLegacyDelegateTwoHandlers {
       XCTAssertEqualObjects(delegate.notificationReceived, _notification);
       delegate.notificationReceived = nil;
 
+      // Verify NOT handling of `application:openURL:sourceApplication:annotation:`.
```

Comment:
Done. Also change all instances of "Verify handling of ..." to be "Verify ... is handled" to be consistent.